src/CMakeLists.txt

```diff
@@ -170,6 +170,10 @@ macro (add_compiler_flag flag)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${flag}")
 endmacro (add_compiler_flag)
 
+# Disallow undefined symbols in object files. Undefined symbols in shared
+# libraries are still allowed.
```

Comment:
Ok, I see. I used something similar here:
http://pkgs.fedoraproject.org/cgit/rpms/gasnet.git/tree/gasnet.spec#n146
but maybe `-z,now` is more what we want.